pysat/tests/test_orbits.py

```diff
@@ -58,56 +58,56 @@ def test_all_single_orbit_calls_in_day(self):
             if i > 14:
                 break
 
-            ans.append(self.testInst.data.index[0] == (pds.datetime(2009,1,1)+i*relativedelta(hours=1, minutes=37)))
-            ans2.append(self.testInst.data.index[-1] == (pds.datetime(2009,1,1)+(i+1)*relativedelta(hours=1, minutes=37) -
+            ans.append(self.testInst.index[0] == (pds.datetime(2009,1,1)+i*relativedelta(hours=1, minutes=37)))
+            ans2.append(self.testInst.index[-1] == (pds.datetime(2009,1,1)+(i+1)*relativedelta(hours=1, minutes=37) -
                                                          relativedelta(seconds=1) ))
 
         assert np.all(ans) & np.all(ans2)
 
     def test_orbit_next_call_no_loaded_data(self):
         self.testInst.orbits.next()
-        ans = (self.testInst.data.index[0] == pds.datetime(2008,1,1))
-        #ans2 = (self.testInst.data.index[-1] == (pds.datetime(2009,1,1,1,36,59) ))
+        ans = (self.testInst.index[0] == pds.datetime(2008,1,1))
+        #ans2 = (self.testInst.index[-1] == (pds.datetime(2009,1,1,1,36,59) ))
```

Comment:
Why is ans2 commented out?

Reply:
Test originated from a full orbit check. This case isn't a full orbit. So I specified the correct times and if things ever change, we'll know.